packages/react-native-bpk-theming/src/util.js

```diff
@@ -119,7 +138,9 @@ export const getThemeAttributes = (
     return null;
   }
 
-  const hasAllRequiredAttributes = isValidTheme(requiredAttributes, theme);
+  const hasAllRequiredAttributes =
```

Comment:
Why are we not using `isValidTheme` here, it seems to do the same?

Reply:
Because `isValidTheme` returns `true` if zero required props are passed in, whereas in this case we *don't* want to pass a theme down to components if there are zero passed in.

Maybe this is confusing and it should be refactored.. what do you think?

Reply:
Got it, makes sense. I guess we can add a comment explaining this as it not that clear